test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveMapValueTranslation.cs

```diff
@@ -206,21 +197,12 @@ public override void OnComponentUpdate(ComponentUpdateOp op)
 
             public override void OnAuthorityChange(AuthorityChangeOp op)
             {
-                if (!IsValidEntityId(op.EntityId, "AuthorityChangeOp", out var entity))
-                {
-                    return;
-                }
-
+                var entity = TryGetEntityFromEntityId(op.EntityId);
                 ApplyAuthorityChange(entity, op.Authority, op.EntityId);
             }
 
             public override void OnCommandRequest(CommandRequestOp op)
             {
```

Comment:
line 228: (for some reason I can' t add a comment on that line) - do we want to change the LogError to an exception / log?

Reply:
Ah yes probably - as its a similar case to an `UnknownComponentId`

Reply:
lgtm :)